packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md

```diff
@@ -102,29 +102,29 @@ Authorization: ${key}
 
 #### Authorization header
 
-The Authorization header (`Authorization: ${key}` is a secret string that you provide to Okta when you register your external service. This string serves as an API access key for your service, and Okta provides it in every request, allowing your code to check for its presence as a security measure. (This isn't an Okta authorization token, it's simply a text string you decide on.) See [HTTP header: Basic Authentication](/docs/guides/common-hook-set-up-steps/#http-header-basic-authentication).
+The Authorization header (`Authorization: ${key}`) is a secret string that you provide to Okta when you register your external service. This string serves as an API access key for your service, and Okta provides it in every request. This string allows your code to check for its presence as a security measure. (This isn't an Okta authorization token, it's simply a text string you decide on.) See [HTTP header: Basic Authentication](/docs/guides/common-hook-set-up-steps/#http-header-basic-authentication).
```

Comment:
Could consider changing "This string allows" to "something like Your code can check if the string is present as a security measure."; just thinking we have two sentences in a row that start with "This string".